src/resources/postcss/components/_search.pcss

```diff
@@ -68,26 +69,82 @@
 			}
 
 			&:first-child {
-				flex: 1 1 34.5%;
+				flex: 1 1 100%;
 			}
 
 			&:nth-child(2) {
-				flex: 1 1 37.5%;
+				flex: 1 1 100%;
 			}
 
-			&:nth-child(3) {
-				flex: 1 1 28%;
+			&:last-child {
+				.tribe-common-c-search__input {
+					@media (--viewport-medium) {
+						border-right: 0;
+					}
+				}
+
 			}
+
 		}
 	}
 
 	.tribe-common-c-search__input {
+
 		@media (--viewport-medium) {
-			border: none;
 			border-right: 1px solid var(--color-border-default);
 			height: auto;
 			padding: 7px 18px 8px;
 		}
+
+		&:focus {
+			outline: none;
+		}
+	}
+
+	.tribe-common-c-search__input--keyword {
+		background-image: svg-inline(search);
+		background-repeat: no-repeat;
+		background-size: 16px;
+		background-position: right center;
+
+		@media (--viewport-medium) {
+			border: none;
+			border-right: 1px solid var(--color-border-default);
+			height: auto;
+			padding: 7px 18px 7px 40px;
+			background-position: 16px center;
+		}
+
+		&:focus {
+			background-image: svg-inline(search-focus);
+		}
+	}
+
+	.tribe-common-c-search__input--keyword-focus {
+		background-image: svg-inline(search-focus);
+	}
+
+	.tribe-common-c-search__input--location {
+		background-image: svg-inline(location);
+		background-repeat: no-repeat;
```

Comment:
lets consolidate some of these common repeated styles for `.tribe-common-c-search__input`

Reply:
These are only for the inputs that have icons. The idea is that `.tribe-common-c-search__input` is the base class for inputs, and this modifiers are specifically for the inputs that came from design (search and location). If somebody (or us as an extension) wants to add a new input, they will use `.tribe-common-c-search__input` and will have the correct input styles there, without icons.

Reply:
same note as https://github.com/moderntribe/tribe-common/pull/1061#discussion_r296099821. I'm also OK with a `.tribe-common-c-search__input--icon` modifier with common styles, but same with the note above, we should keep as much common as possible until a new design forces us to split off specifics.